site/zh/blog/new-in-devtools-99/index.md

```diff
@@ -6,17 +6,16 @@ authors:
 date: 2022-02-21
 updated: 2022-02-21
 description:
-  "Throttling WebSocket requests, new Reporting API pane, console styling and more."
+  "WebSocket 限速，Reporting API 新边栏，控制台美化以及更多。"
```

Comment:
1. Should we use `、` (顿号) instead of `,` here? I remember we used `、` previously.
2. Should we translate `throttling` to `节流` instead? I remember it's more conventional.
3. Maybe also translate `requests` to become `WebSocket 请求节流`.

Reply:
> 1. Should we use 、 (顿号) instead of , here? I remember we used 、 previously

Yep, you use `、` in description section of a certain translations(like `new-in-devtools-92`, `new-in-devtools-95` and `new-in-devtools-98`). I use `,` in other translations instead. So we should keep consistent style with each other. @chybie 

> 2. Maybe also translate requests to become WebSocket 请求限速

`请求` seems to be a verb. So I get ride of it from the resulting translation.

Reply:
I think "请求" is quite important in this context here. If you think "WebSocket 请求限速" makes "请求" sound like a verb, we can use "WebSocket 请求的限速" to avoid ambiguity.

Reply:
That sounds good. But I think `限速 WebSocket 请求` might be a better choice.

Reply:
yep, this sounds better.